ci/docker-publish.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         if: github.event_name != 'pull_request'
         uses: sigstore/cosign-installer@d6a3abf1bdea83574e28d40543793018b6035605
```

Comment:
Does this version of `cosign-installer` allows the below mentioned `cosign-release` version?

Reply:
Yep, it should!